NuRadioReco/detector/detector_base.py

```diff
@@ -967,10 +967,10 @@ def get_channel_group_id(self, station_id, channel_id):
         """
         res = self.__get_channel(station_id, channel_id)
         if 'channel_group_id' not in res.keys():
-            logger.warning(
-                'Channel group ID not set for channel {} in station {}, returning -1'.format(
+            logger.info(
```

Comment:
I agree with you that warning is a bit much, but maybe info is too little? Not sure. I think STATUS would be optimal actually, as that would still show up in the logs by default in the new system. Though for that we need to wait until the new logging is done :)